src/vstest.console/Processors/EnableBlameArgumentProcessor.cs

```diff
@@ -133,7 +136,8 @@ public void Initialize(string argument)
         {
             bool isDumpEnabled = false;
 
-            if (!string.IsNullOrWhiteSpace(argument) && argument.Equals(Constants.BlameCollectDumpKey, StringComparison.OrdinalIgnoreCase))
+            var parseSucceeded = LoggerUtilities.TryParseLoggerArgument(argument, out string loggerIdentifier, out Dictionary<string, string> parameters);
```

Comment:
So it seems like now `/blame:collect;DumpType=full` is the way to collect full dump.
In platform terminology we use arg like `/argName:argidentifier;key1=value1`. This makes `collect` as identifier of blame arg which seem correct. @cltshivash @singhsarab @kaadhina Any suggestions here?